src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -502,6 +525,16 @@ protected function save_widget( $request ) {
 			$widget_object->updated = false;
 		}
 
+		/**
+		 * Fires after a widget is completely created or updated via the REST API.
+		 *
+		 * @since 5.8.0
+		 * @param WP_Widget       $widget_object Inserted or updated widget object.
+		 * @param WP_REST_Request $request       Request object.
+		 * @param bool            $creating      True when creating a post, false when updating.
+		 */
+		do_action( 'rest_after_save_widget', $widget_object, $request, $creating );
```

Comment:
Same here about passing `$widget_id` and `$sidebar_id` instead of `$widget_object`.